### PR TITLE
add connectComponentsPlanner breaking change

### PR DESCRIPTION
**Pull request by MacondoExpress**
Add GraphQL context breaking changes

### CODE REVIEW

File: modules/ROOT/pages/migration/index.adoc [hunk 1]
```diff
@@ -321,6 +321,11 @@ The following `@cypher` field types have been removed from the `sort` argument b
   - `@cypher` fields with non-sortable type like `Node`, `List` etc.
 
 
+=== Removed `connectComponentsPlanner` from `cypherQueryOptions`
+
+Following the deprecation of the `connectComponentsPlanner` https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/query-tuning/#cypher-connect-components-planner, 
+the field `connectComponentsPlanner` is no longer accepted in the Context and is removed from the `cypherQueryOptions` object.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
Following the deprecation of the link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/query-tuning/#cypher-connect-components-planner[`connectComponentsPlanner`], the field is no longer accepted in the context and is removed from the `cypherQueryOptions` object.
```

what does "context" refer to here?

Comment (MacondoExpress) [reply modules/ROOT/pages/migration/index.adoc]: Is the GraphQL context passed in the GraphQL resolver

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-215.surge.sh

Updated pages:
- [Migration to 6.0.0](https://neo4j-docs-graphql-215.surge.sh/graphql/6/migration)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by rsill-neo4j — APPROVED** [review]

**Pull request merged**